bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/component.jsx

```diff
@@ -666,24 +686,30 @@ class PresentationUploader extends Component {
     );
   }
 
-  handleSendToChat(item) {
+  handleDownloadingOfPresentation(item, type) {
     const {
-      exportPresentationToChat,
+      exportPresentation,
       intl,
     } = this.props;
 
     const observer = (exportation, stopped) => {
       this.deepMergeUpdateFileKey(item.id, 'exportation', exportation);
 
       if (exportation.status === EXPORT_STATUSES.EXPORTED && stopped) {
```

Comment:
This `stopped` is `false` when the PDF export with annotations completes, so the notification for the `"Annotated"` job type is not shown until an `"Original"` one comes in. See previous video

Reply:
Thank's for pointing that out, https://github.com/bigbluebutton/bigbluebutton/pull/17905 resolves the problem mentioned:
> If you export a file with annotations say 5 times, and then also enable the original download, 6 notifications...